builder/words

```diff
@@ -0,0 +1,42 @@
+hello                    |     hello
```

Comment:
No, its a build-time thing but its not a parser thing, its more like a cmd interpreter thing - so its after the parser.